controllers/topics.js

```diff
@@ -15,7 +15,6 @@ const router = express.Router({ mergeParams: true });
 const {
 	NEXBOARD_USER_ID,
 	NEXBOARD_API_KEY,
-	PUBLIC_BACKEND_URL,
```

Comment:
What is the relation from this to the Ticket?

Reply:
it's another occurrence of same env which was checked to be updated too, however it was not in use and was cleanued up.